src/test/scala/org/broadinstitute/dsde/workbench/sam/api/AdminServiceUserRoutesSpec.scala

```diff
@@ -118,6 +118,7 @@ class AdminServiceUserRoutesSpec extends AnyFlatSpec with Matchers with Scalates
     // Arrange
     val samRoutes = new MockSamRoutesBuilder(allUsersGroup)
       .withEnabledUser(defaultUser)
+      .withAllowedUser(defaultUser)
```

Comment:
We have a distinction in sam between a user being 'allowed' to use the system (accepted tos for example) and a user being 'enabled' (a flag on the user database model)

Reply:
This is interesting - I find the phrase `allowed` to be confusing in that an allow state could, in theory, apply to a LOT of different things like `allowed as an admin` or `allowed as a reader` etc. As opposed to something like `withToSAccepted()` or  `withToSRejected()` which is much more specific to the feature you're describing here.